docs/guides/events/basics.md

```diff
@@ -0,0 +1,84 @@
+# Basics
+
+```{attention}
+This guide only applies to GatewayBot. REST-only applications cannot receive events through the gateway.
+```
+
+## What are events?
+
+When your application connects to the Discord Gateway, it will start receiveing information about actions
+happening within the guilds your bot is a member of. These pieces of information are called "events",
+and inform your bot about actions other users or bots may have triggered.
+
+An example would be `MessageCreateEvent`, which the bot receives every time a message is sent in a channel
+the bot can see. It contains some information about the message, where it was sent, etc..
+
+## Listeners
+
+To execute code when the bot receives an event, we can create a listener. This is an async function that
+will be called every time an event of the type we specified is encountered.
+
+```py
+@bot.listen()
+async def message_listener(event: hikari.MessageCreateEvent) -> None:
+    print(f"I have received a message from {event.author} in {event.channel_id}!")
+```
+
+You may also put the event's type in the decorator instead of a type annotation:
+
+```py
+@bot.listen(hikari.MessageCreateEvent)
+async def message_listener(event) -> None:
+    print(f"I have received a message from {event.author} in {event.channel_id}!")
+```
```

Comment:
Although this is possible, does the guide need to show this?

```suggestion
You may also put the event's type in the decorator instead of a type annotation:

```py
@bot.listen(hikari.MessageCreateEvent)
async def message_listener(event) -> None:
    print(f"I have received a message from {event.author} in {event.channel_id}!")
```
```

Reply:
I think it is important to note in scenarios where type annotation usage may not be feasible or possible, yes.